### PR TITLE
Update resource descriptor file for OMIM to MIM change

### PR DESCRIPTION
**Pull request by chris-grove**
I've updated the OMIM stanza in the resource descriptors YAML file to accommodate the change in curie prefix from "OMIM" to "MIM". I think I made the changes that are needed, but please have a look to check.

### CODE REVIEW

**Comment by chris-grove** [comment]
@sbello I updated the phenotypic series example ID because the existing one appears to be invalid at OMIM

File: resourceDescriptors.yaml [hunk 1]
```diff
@@ -687,11 +687,11 @@
...
       url: https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=[%s]
 
-- db_prefix: OMIM
-  name: OMIM
-  aliases: ['omim']
-  example_id: OMIM:600483, OMIM:PS100200
-  gid_pattern: "^(OMIM|omim):[PS]*\\d+$"
+- db_prefix: MIM
+  name: MIM
+  aliases: ['mim']
```
Comment (sierra-moxon) [inline resourceDescriptors.yaml:692]: do you want to add an alias for "OMIM" as well here?

Comment (chris-grove) [reply resourceDescriptors.yaml:692]: @sierra-moxon You mean have:
`aliases: ['mim', 'omim']`
? It was omim and I changed it to mim. Adding 'omim' back is fine with me, I'm not clear on how that works.

Comment (oblodgett) [reply resourceDescriptors.yaml:692]: I don't think we make use of the aliases but might be good to have both of them.

**Review by oblodgett — APPROVED** [review]

**Comment by chris-grove** [comment]
OK, thanks @sierra-moxon and @oblodgett 
I've added the 'omim' alias back

File: resourceDescriptors.yaml (hunk 1 above)
Comment (sbello) [reply resourceDescriptors.yaml]: Looks good to me. I agree that having both MIM and OMIM as aliases would be good. 
Have we checked to make sure we aren't bringing in OMIM IDs from any other sources? I don't see them on the gene pages but I haven't checked all the loads.

**Review by sbello — APPROVED** [review]
Looks good

**Pull request merged**